docs/book_files/csi_block_storage_kc_rn.md

```diff
@@ -0,0 +1,4 @@
+# Release notes
```

Comment:
why is this under `book_files` and not `content/release_notes`?

Reply:
it's an IBM only topic to introduce the information. it's also not in the PDF

Reply:
is `book_files` for PDF only?
if so, could we move it to a different dir?

Reply:
@oriyarde  as I mentioned in our slack conversation - this is for IBM Docs content and cannot be moved.